drivers/wdm/audio/backpln/portcls/dll.cpp

```diff
@@ -6,31 +6,23 @@
  * PROGRAMMER:      Andrew Greenwood
  */
 
-#include "private.hpp"
-
-#ifndef YDEBUG
-#define NDEBUG
-#endif
 
-#include <debug.h>
+#include "private.hpp"
```

Comment:
strip one newline above #include "private.hpp" as it will help git compare to better align what should be aligned.

Reply:
Fixed.